extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -159,6 +159,17 @@ export class AmpSlideScroll extends BaseSlides {
       this.slidesContainer_.addEventListener(
           'touchend', this.touchEndHandler_.bind(this));
     }
+
+    this.registerAction('moveToSlideIndex', invocation => {
+      const args = invocation.args;
+      if (!args) {
+        return;
+      }
+      let slideIndex = Number(args['newIndex']);
```

Comment:
trying to be consistent with rest of the file (naming it slide can be super confusing in the slide.)